pkg/swagger/typewalker.go

```diff
@@ -92,6 +92,7 @@ func (tw *typeWalker) schemaFromType(t types.Type, deps map[*types.Named]struct{
 			s.Type = "boolean"
 		case types.Int:
 			s.Type = "integer"
+			s.Format = "int32"
```

Comment:
Is this a breaking change? This seems to impact the stable API too, not just the preview one

Reply:
This was indeed flagged as a breaking change - but the requirement of the int32 format on all integers was listed as a "must-fix" on the CI during the upstream process. As such this must-fix introduces the "breaking change" but we can't get approval to upstream without fixing the must fix. One thing to also consider is that we also only define bools, ints and strings once on our side in the generator code - for all iterations of the aforementioned. Definitely open to suggestions on this one.